frontend/src/features/banners/WebsiteBanner.tsx

```diff
@@ -67,14 +72,25 @@ const WebsiteBanner = ({ banner }: WebsiteBannerSingleProps) => {
               justifyContent="space-between"
               alignItems="flex-end"
             >
-              <Typography paragraph mb={0}>
-                {banner.text}
-              </Typography>
+              <Box>
+                {bannerParagraphs.map((paragraph, index) => (
+                  <Typography
+                    paragraph
+                    key={`banner_p${index}`}
+                    mb={index + 1 === bannerParagraphs.length ? 0 : 2}
```

Comment:
Instead of creating multiple paragraphs, it may be easier and more flexible to use   **`white-space: pre-wrap`**  to preserve all spaces and new lines from the source, just like we do for the video description:

https://github.com/tournesol-app/tournesol/blob/9d25186cfe8da5e3de56b6ad02105d8652486136/frontend/src/pages/videos/VideoAnalysisPage.tsx#L73-L84